tests/e2e/e2e_test.go

```diff
@@ -623,27 +587,11 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 			cancel()
 			gomega.Ω(err).Should(gomega.BeNil())
 		})
-		ginkgo.By("wait for network to be healthy", func() {
+		ginkgo.By("wait for the network to be healthy", func() {
 			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-			var created bool
-			continueLoop := true
-			for continueLoop {
-				select {
-				case <-ctx.Done():
-					continueLoop = false
-				case <-time.After(5 * time.Second):
-					ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
-					status, err := cli.Status(ctx)
-					cancel()
-					gomega.Ω(err).Should(gomega.BeNil())
-					created = status.ClusterInfo.CustomChainsHealthy
-					if created {
-						continueLoop = false
-					}
-				}
-			}
+			_, err := cli.Health(ctx)
```

Comment:
I think either there should be an additional `WaitUntilHealthy` method server side, or we should keep this the responsibility of the client to wait until the network is healthy. I am not sure if being this hard on when to return `Health` is appropriate